static/scss/positions.scss

```diff
@@ -1,4 +1,4 @@
-@quote-size: 1em;
+$quote-size: 1em;
```

Comment:
This is *the* main syntax change from LESS to SCSS (given what we use of LESS anyways). All other existing LESS code is unchanged.